File: go/src/services/payment/paypal_service.go
package payment

import (
	"fmt"

	"github.com/fatih/color"
)

type paypalService struct {
}

func (service *paypalService) pay(id string, amount float64) {
	color.HiYellow("-- Paypal Service [start] --")
	fmt.Println()

	result := fmt.Sprintf("Payment processed: [%s] $%.2f", id, amount)

	fmt.Println(result)

	fmt.Println()
	color.HiYellow("-- Paypal Service [end] --")
}

File: ts/src/services/paypal-service.ts
export class PaypalService {
  public pay(id: string, amount: number): void {
    console.log("-- Paypal Service [start] --");

    const result = `Payment processed: [${id}] $${amount}`;
    console.log(result);

    console.log("-- Paypal Service [end] --");
  }
}
